duckduckgo/iOS
# Convert all autofill strings to localised strings

Task/Issue URL: https://app.asana.com/0/0/1202809065048070/f
Tech Design URL:
CC:

**Description**:
The worlds most tedious PR(tm)!
As per title, also will add translations (at time of writing translations not ready)




**Steps to test this PR**:
1. Use autofill and check that everything looks correct, pay special attention to any strings that had params, since they were easier to mess up
2. Test some other languages, especially the longer ones to check they are okay



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
